src/app/complex/complex-results/complex-navigator/complex-list-display-buttons/complex-list-display-buttons.component.html

```diff
@@ -1,11 +1,11 @@
 <div class="displayButton columns medium-12" [ngClass]="">
-  <button type="submit" name="submit" class="button view-button"
-          (click)="setListView()" [ngClass]="{'selected': !isDisplayComplexNavigatorView()}">
+  <button class="button view-button"
+          (click)="setListView()" [class.selected]="displayType() === 'view_list'">
     View list
   </button>
-  <button type="submit" name="submit" class="button view-button cnButton"
-          (click)="setComplexNavigatorView()" [ngClass]="{'selected': isDisplayComplexNavigatorView()}">
+  <button class="button view-button cnButton"
+          (click)="setComplexNavigatorView()" [class.selected]="displayType() === 'view_complex_navigator'">
```

Comment:
The point of using the function `isDisplayComplexNavigatorView` is to only define the string in a single place and then rely in the const. Now the `view_complex_navigator` is in two places in the HTML and two places in the ts file, so there's more changes to have a typo, and if we update it, more changes to miss one of the places to change it.

Reply:
Fair, though I introduced a type restriction which is stronger than a recommended const usage. we can probably mix both by using an enum instead